taxi/tests/test_views_manufacturer.py

```diff
@@ -0,0 +1,94 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.forms import ManufacturerSearchForm
+from taxi.models import Manufacturer
+
+
+class PublicManufacturerlistTest(TestCase):
+    def test_manufacturer_list(self):
+        res = self.client.get(
+            reverse("taxi:manufacturer-list")
+        )
+        self.assertNotEqual(res.status_code, 200)
+
+    def test_manufacturer_update_login_required(self):
+        url = reverse("taxi:manufacturer-update", kwargs={"pk": 1})
+        response = self.client.get(url)
+        self.assertNotEqual(response.status_code, 200)
+
+    def test_manufacturer_create_login_required(self):
+        url = reverse("taxi:manufacturer-create")
+        response = self.client.get(url)
+        self.assertNotEqual(response.status_code, 200)
+
+    def test_manufacturer_delete_login_required(self):
+        manufacturer = Manufacturer.objects.create(
+            name="Test Manufacturer", country="Test Country"
+        )
+        url = reverse(
+            "taxi:manufacturer-delete",
+            kwargs={"pk": manufacturer.id}
+        )
+        response = self.client.get(url)
+        self.assertNotEqual(response.status_code, 200)
+
+
+class PrivateAlllistTest(TestCase):
+    def setUp(self) -> None:
+        self.manufacturer1 = Manufacturer.objects.create(
+            name="Test Manufacturer", country="Test Country"
+        )
+        self.manufacturer2 = Manufacturer.objects.create(
+            name="Another Manufacturer", country="Test Country2"
+        )
+        self.driver = get_user_model().objects.create_user(
+            username="test",
+            password="test1234",
+        )
+        self.client.force_login(self.driver)
+
+    def test_manufacturer_list(self):
+        Manufacturer.objects.create(
+            name="test",
+            country="Ukraine",
+        )
+        Manufacturer.objects.create(
+            name="test_china",
+            country="China",
+        )
+        respons = self.client.get(reverse("taxi:manufacturer-list"))
+        self.assertEqual(respons.status_code, 200)
+
+    def test_manufacturer_search_list(self):
+        Manufacturer.objects.create(
+            name="test",
+            country="Ukraine",
+        )
+        Manufacturer.objects.create(
+            name="test_china",
+            country="China",
+        )
+        form = ManufacturerSearchForm(data={"name": self.manufacturer1.name})
+        respons = self.client.get(reverse("taxi:manufacturer-list"))

```

Comment:
There is a typo in the variable name `respons`. It should be `response` to maintain consistency and avoid confusion.